### ISSUE TITLE
not working in Windows

### ISSUE THREAD

**Issue by ilyapashuk** [mrsarm/jshell-plugin#10]
hello

plugin is not working in Windows.

jshell is started, but there is no my classes around.

In windows, if classpath is sent as a one string, multiple dirs and jars are separated by the ; char, not by : char as in linux.

is this handled correctly in your plugin?

**Comment by ilyapashuk** [comment]
now I had tested in arch linux, not working too.

Java 17 and gradle 7.3.2 in both systems

**Comment by ilyapashuk** [comment]
my simple project is in attachment
[simpledb.zip](https://github.com/mrsarm/jshell-plugin/files/7792707/simpledb.zip)

I used java library plugin type.

**Comment by mrsarm** [comment]
Hi @ilyapashuk ! Thanks for the sample, that will help me understand the problem better.

Talk to you here soon.

**Comment by ilyapashuk** [comment]
it's better to use .pam_environment or /etc/environment because with 
.bashrc or .profile accessibility will be only enabled for applications 
launched through a terminal.


so you will always need to launch your browser, electron app or skype by 
terminal.


with .pam_environment it will be enabled globally and will simply work.

**Comment by ilyapashuk** [comment]
as I can sea, jshell task depends on classes task to build classes before running jshell.

maybe it will be better to build full jar instead of only classes?

with full jar resources will work fyne, which may not be the case with only classes.

**Comment by mrsarm** [comment]
> it's better to use .pam_environment or /etc/environment because with...

Not sure how this is related @ilyapashuk 

> maybe it will be better to build full jar instead of only classes?

I tried executing first `gradle assemble` but got the same results. Did you try something different?

**Comment by mrsarm** [comment]
@ilyapashuk I just added some workarounds to improve the classpath detection, please checkout the pre-release 1.2.0-RC1 and execute again (in fact, I have checked the workaround with your project and indeed did work, but I want to check that it also works from your environment).

```groovy
plugins {
    // Apply the java-library plugin for API and implementation separation.
    id 'java-library'
    id "com.github.mrsarm.jshell.plugin" version "1.2.0-RC1"
}
```

**Comment by ilyapashuk** [comment]
now it works, but shows warning about using of deprecated gradle features, which will not be available in gradle 8.0

**Comment by mrsarm** [comment]
Good catch @ilyapashuk ! your feedback is very valuable, I didn't notice that warning, now is fixed, and should be ready for Gradle 8.

Please test the version `1.2.0-RC2` whenever you can.

**Comment by ilyapashuk** [comment]
Now it works without warnings, thank you.

**Comment by mrsarm** [comment]
Great ! I'll merge it soon.

**Comment by mrsarm** [comment]
Fixed in version [1.2.0](https://github.com/mrsarm/jshell-plugin/releases/tag/1.2.0).
